Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeAdapter.kt

```diff
@@ -42,6 +43,7 @@ class HomeAdapter :
                 StableIdMod(data),
                 DataBinderMod<HomeItem, HomeItemVH<HomeItem, ViewBinding>>(data),
                 TypedVHCreatorMod({ data[it] is FAQCard.Item }) { FAQCard(it) },
+                TypedVHCreatorMod({ data[it] is CreateEventCard.Item }) { CreateEventCard(it) },
```

Comment:
do we call it `Event` or `TraceLocation`? could you please double check how is it called in the organiser flow ?

Reply:
I don't see any mention of tracelocation in figma, ticket or tech spec. I'm not sure if it's been decided we'll use TraceLocation instead of Event but imo TraceLocation doesn't tell me anything. CreateEventCard sounds more explicit in this case but if it's unanimously decided by the team, I'll change it. Same for the rest of your comments relating to the naming convention.

Reply:
Could you please align with @LukasLechnerDev?  I think he is using `TraceLocation` already, or?

Reply:
`TraceLocation` is used a lot in the TechSpec, e.g. [here](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/event-registration-mvp/docs/spec/event-registration-client.md#organizer-tracelocation-creation) and we mostly use this term instead of event. 

As far as I know the term `Event` was used at the beginning but was later changed to `TraceLocation`, because you not only create QR-codes for "Events" but also "Locations" (Without start and end date). Therefore, we mostly use `TraceLocation` and we differentiate by a "permanent" `TraceLocation` (location) and a "temporary" `TraceLocation` (event)

Reply:
Ok, now I get the naming. I'll use TraceLocation then but in all honesty, for somebody first looking through the code, reading TraceLocation would make you think it's related to some GPS tracking stuff and not really events and physical locations.